docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -105,6 +108,15 @@ must be mounted as a
 in the plugin's
 [PodSpec](/docs/api-reference/{{page.version}}/#podspec-v1-core).
 
+As Kubernetes device plugin support is still in alpha stage, its API version can be upgraded in
```

Comment:
Replace this paragraph with:
```
Kubernetes device plugin support is still in alpha. As development continues, its API version can change in incompatible ways. We recommend that device plugin developers do the following:
- Watch for changes in future releases
- Support multiple versions of the device plugin API for backward/forward compatibility

If you enable the DevicePlugins feature and run multiple device plugins requiring different Kubernetes version support, upgrade your device plugins to support both versions before upgrading these nodes to ensure the continuous functioning of the device allocations during the upgrade.
```
So:
> Kubernetes device plugin support is still in alpha. As development continues, its API version can change in incompatible ways. We recommend that device plugin developers do the following:
> - Watch for changes in future releases
> - Support multiple versions of the device plugin API for backward/forward compatibility
>
> If you enable the DevicePlugins feature and run multiple device plugins requiring different Kubernetes version support, upgrade your device plugins to support both versions before upgrading these nodes to ensure the continuous functioning of the device allocations during the upgrade.

Reply:
Thanks for the suggestion! I modified it a bit. Instead of saying "If you enable the DevicePlugins feature and run multiple device plugins requiring different Kubernetes version support", I changed to say "If you enable the DevicePlugins feature and run device plugins on nodes that need to be upgraded to a Kubernetes release with a newer device plugin API version". The reason that a device plugin may need to support multiple versions is because it needs to talk to Kubelet that may uses a different version of API.

Reply:
That works. :+1:

Reply:
I’ll give this a docs LGTM when I get back to my laptop.